### ISSUE TITLE
TypeError: 'ListTank' object does not support indexing

### ISSUE THREAD

**Issue by retrosnob** [faulknerrainford/MetaChem#2]
Running stringcat_graph.py fails at this line because containersin is an object of type ListTank and does not support indexing. Should containersin be a list of ListTanks?

![image](https://user-images.githubusercontent.com/9262719/64469296-f9884100-d161-11e9-8ab3-0c40b447375e.png)

**Comment by faulknerrainford** [comment]
Yes, this is a transfer function so it takes a list of tanks as containers in and out so it can move things between them.

**Comment by retrosnob** [comment]
Except in the code a single ListTank object is passed in rather than a list. So containersin[0] generates an error since a single ListTank object is being indexed instead of a list of ListTanks. Hence ListTank object does not support indexing. 

I've forked your code and I'm porting it to Python3. With your permission I might fire you some questions about it as they come up.

**Comment by faulknerrainford** [comment]
Looking at it, that doesn't need the index (I probably just did a copy
paste at some point, I don't remember testing all that carefully on this
node). The current set up for it is that the ListTank is set up with
multiple tanks so it should only be reading in from a single list tank node.

Feel free to fire questions my way. I'll try to answer in good time.
